packages/app-store/salesforce/components/EventTypeAppCardInterface.tsx

```diff
@@ -36,7 +36,13 @@ const EventTypeAppCard: EventTypeAppCardComponent = function EventTypeAppCard({
               label={t("skip_rr_assignment_label")}
               labelOnLeading
               checked={isRoundRobinLeadSkipEnabled}
-              onCheckedChange={(checked) => setAppData("roundRobinLeadSkip", checked)}
+              onCheckedChange={(checked) => {
+                setAppData("roundRobinLeadSkip", checked);
+                if (checked) {
+                  // temporary solution, enabled should always be already set
```

Comment:
There is an issue with the event metadata: https://github.com/calcom/cal.com/issues/15169

This at least makes sure that when `roundRobinLeadSkip` is turned on, the enabled key will be added in case it is missing